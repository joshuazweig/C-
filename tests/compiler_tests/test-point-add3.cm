int main() {
    stone c;
    stone d;

    stone a1;
    stone b1;
    stone p;

    c = "19";
    d = "8";

    a1 = "7";
    b1 = "26";
    p = "61";

    mint A;
    mint B;
    curve *E;

    A = <a1, p>;
    B = <b1, p>;
    E = <A, B>;

    print_curve(E);

    point *Q;
    point *R;

    Q = <E, c, d>;

    d = "53";
    R = <E, c, d>;

    print_point(Q+R); // should print inf
    return 0;
}
