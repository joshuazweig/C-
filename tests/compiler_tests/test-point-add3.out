a: 7
b: 26
p: 61
inf
